server/src/test/java/io/crate/integrationtests/SelectOrderByIntegrationTest.java

```diff
@@ -186,4 +186,16 @@ public void testOrderByLiteralConstant() {
         assertThat(printedTable(response.rows()), Matchers.is("1| 1\n" +
                                                               "1| 2\n"));
     }
+
+    @Test
+    public void testOrderByAliasedLiteral() {
+        execute("create table t1 (id int)");
+        execute("insert into t1 (id) values (1), (2)");
+        refresh();
+        execute("select 'a' as n1, 12 as n2, id as n3 from t1 order by 2, 3, 1");
+        assertThat(printedTable(response.rows()), Matchers.is("""
+                                                                  a| 12| 1
+                                                                  a| 12| 2
+                                                                  """));
```

Comment:
Can we convert this into a `CollectTest` unit test?
See e.g. https://github.com/crate/crate/blob/master/server/src/test/java/io/crate/planner/operators/JoinTest.java#L417 on how to build and test an execution plan.

Reply:
I am looking into it now, Thanks!

Reply:
Hi Basti, checked in the change, please check once again.